tests/spec/fs.rename.spec.js

```diff
@@ -10,6 +10,39 @@ describe('fs.rename', function() {
     expect(fs.rename).to.be.a('function');
   });
 
+  it('should not change an existing file to itself', function(done) {
```

Comment:
Try saying what it _should do_ rather than what it _shouldn't do_. I think that might help make it clearer what the indented behaviour is.